src/proof/dot/dot_printer.h

```diff
@@ -58,9 +97,28 @@ class DotPrinter
   uint64_t printInternal(std::ostream& out,
                          const ProofNode* pn,
                          std::map<size_t, uint64_t>& pfLet,
+                         NodeClusterType parentType,
                          uint64_t scopeCounter,
                          bool inPropositionalView);
 
+  /**
+   * Print the nodes of the proof in the format:
+   * $NODE_ID [ label = "{$CONCLUSION|$RULE_NAME($RULE_ARGUMENTS)}",
+   * $COLORS_AND_CLASSES_RELATED_TO_THE_RULE ];
+   * @param out the output stream
+   * @param pn the proof node to print
+   * @param currentRuleID the current rule ID
+   * @param scopeCounter counter of how many SCOPE were already depth-first
+   * traversed in the proof up to this point
+   * @param inPropositionalView flag used to mark the proof node being traversed
+   * was generated by the SAT solver and thus belong to the propositional view
+   */
+  inline void printNodeInfo(std::ostream& out,
+                            const ProofNode* pn,
+                            uint64_t currentRuleID,
+                            uint64_t& scopeCounter,
+                            bool& inPropositionalView);
```

Comment:
Rename this guy to better reflect that we are not talking about "views" anymore but about proof node cluster types. Update the above method as well.